src/current/v23.1/table-localities.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Table Localities
+summary: Learn how to use table localities in a CockroachDB multi-region cluster.
+toc: true
+keywords: gin, gin index, gin indexes, inverted index, inverted indexes, accelerated index, accelerated indexes
+docs_area: deploy
+---
```

Comment:
```suggestion
key: when-to-use-regional-vs-global-tables.html
---
```

This ensures the version switcher is able to go back and forth between the old page name and new page name.